PrebidMobile/PrebidMobile-core/src/main/java/org/prebid/mobile/Util.java

```diff
@@ -55,7 +53,9 @@
     static final String MOPUB_BANNER_VIEW_CLASS = "com.mopub.mobileads.MoPubView";
     static final String MOPUB_INTERSTITIAL_CLASS = "com.mopub.mobileads.MoPubInterstitial";
     static final String AD_MANAGER_REQUEST_CLASS = "com.google.android.gms.ads.doubleclick.PublisherAdRequest";
+    static final String AD_MANAGER_REQUEST_CLASS_2 = "com.google.android.gms.ads.admanager.AdManagerAdRequest";
```

Comment:
It would be nice to have some more informative name instead of adding `_2` postfix to all newly introduced class names. Maybe adding a LEGACY_ prefix to old names or adding mentions about V20 support in new names would be a better solution? Because after removing old version support we will end up with _2 postfix which will be not relevant anymore.

Reply:
done